streamalert/alert_processor/outputs/aws.py

```diff
@@ -566,7 +566,8 @@ def _build_email(cls, alert, publication, creds):
         # Attach the record to the email
         if publication.get("@aws-ses.attach_record", True):
             record = json.dumps(alert.record, sort_keys=True, indent=2)
-            msg = cls._add_attachment(msg, "record.json", record)
+            record_name = publication.get("@aws-ses.record_name", "record")
```

Comment:
in line with how other "defaults" are configured. I would move `"record"` to `default_record_name = "record"` and then reference this on this line (similar to the default_body etc)